tests/plugins/test_docker_api.py

```diff
@@ -17,14 +17,22 @@
 from atomic_reactor.build import InsideBuilder, BuildResult
 from atomic_reactor.util import ImageName, CommandResult
 from atomic_reactor.inner import DockerBuildWorkflow
+from atomic_reactor.constants import INSPECT_ROOTFS
 
 from tests.docker_mock import mock_docker
 from flexmock import flexmock
 import pytest
 from tests.constants import MOCK_SOURCE
 
 
+class MockDocker(object):
+    def history(self, name):
+        return []
```

Comment:
Don't we want to include some mocked data here, to test that we get the order right?

Reply:
Oh, this is in the generic test machinery, not in the testing for layer sizes, I get it now.

Reply:
yeah test for layer sizes is in test_inner